src/lambda_function/dispatch/README.md

```diff
@@ -0,0 +1,35 @@
+# Dispatch Lambda
+
+The dispatch Lambda polls the input-to-dispatch SQS queue and publishes to the dispatch SNS topic.
+Its purpose is to inspect an export and dispatch each file as a separate job to eventually be consumed
+by the dispatch-to-raw Lambda.
```

Comment:
The way I wrote this is a bit ambiguous, but since our Lambdas are the only things doing any work and SNS/SQS is just a hook I want to communicate that the point of dispatching these jobs is so that they are taken up by the dispatch to raw Lambda (hence "eventually").

Reply:
Yes I see what you mean now. We have a dispatch lambda and a to-be implemented Dispatch to Raw Lambda. The names were confusing me.